gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/DagManagerTest.java

```diff
@@ -736,6 +736,9 @@ public void testJobStartSLAKilledDag() throws URISyntaxException, IOException {
     Assert.assertEquals(this.dagToJobs.size(), 1);
     Assert.assertTrue(this.dags.containsKey(dagId1));
 
+    String slakilledMeterName = MetricRegistry.name(ServiceMetricNames.GOBBLIN_SERVICE_PREFIX, "job0", ServiceMetricNames.START_SLA_EXCEEDED_FLOWS_METER);
+    Assert.assertEquals(metricContext.getParent().get().getMeters().get(slakilledMeterName).getCount(), 1);
```

Comment:
would it be worth verifying that meters are per-executor, not a single aggregation?